barman/output.py

```diff
@@ -709,17 +709,50 @@ def render_show_backup_general(backup_info, output_fun, row):
         :param str row: format string which allows for `key: value` rows to be
             formatted
         """
-        if "backup_name" in backup_info and backup_info["backup_name"] is not None:
-            output_fun(row.format("Backup Name", backup_info["backup_name"]))
+        backup_name = backup_info.get("backup_name")
+        if backup_name:
+            output_fun(row.format("Backup Name", backup_name))
+
         output_fun(row.format("Server Name", backup_info["server_name"]))
-        if backup_info["systemid"]:
+
+        backup_name = backup_info.get("systemid")
+        if backup_name:
```

Comment:
```suggestion
        system_id = backup_info.get("systemid")
        if system_id:
```

We are reusing a variable with a confusing name :)